Meta/run.py

```diff
@@ -689,7 +689,7 @@ def set_up_network_hardware(config: Configuration):
 
 def set_up_kernel(config: Configuration):
     if config.architecture == Arch.Aarch64:
-        config.kernel_and_initrd_arguments = ["-kernel", "Kernel/Kernel"]
```

Comment:
just asked this on the riscv PR but would adding a linux boot header let us avoid this?

Reply:
I have no clue, better ask in #aarch64/#kernel/#riscv

Reply:
I don't think that a linux header would help here.
The linux header would be at the start of the kernel binary, but `Kernel` is an ELF file.

The header is only needed on RISC-V so we can use booti to boot the kernel. `go` doesn't give us an FDT on RISC-V. This likely also applies to aarch64, but we currently only use the Raspberry Pi bootloader so we don't need a linux header for now.
I think that we should remove that header from the RISC-V kernel once the EFI prekernel works good enough. We would still be able to boot the kernel via U-Boot as you can use `bootefi` to boot EFI executables.